app/adapters/parameter-alias.js

```diff
@@ -0,0 +1,29 @@
+import DS from 'ember-data';
+import ENV from '../config/environment';
+
+const { JSONAPIAdapter } = DS;
+
+export default JSONAPIAdapter.extend({
+
+    caxe: Ember.inject.service(),
```

Comment:
Case service maintains what the currently active case is

Reply:
this is rad, just make this evident in the code too - could you call it activeCase or something? I'm also not entirely sure what `caxe` does.